### PR TITLE
TRUNK-5778: Upgrade com.thoughtworks.xstream 

### PR DESCRIPTION
**Pull request by achilep**
## Description of what I changed

The idea is to  upgrade the com.thoughtworks.xstream to the latest version .
com.thoughtworks.xstream:xstream:1.4.11.1 --> 1.4.12
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5778

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -371,7 +371,7 @@
 			<dependency>
 				<groupId>com.thoughtworks.xstream</groupId>
 				<artifactId>xstream</artifactId>
-				<version>1.4.11.1</version>
+				<version>1.4.12</version>
```
Comment (HerbertYiga) [inline pom.xml:374]: try out the changes with reference application

Comment (achilep) [reply pom.xml:374]: thanks @ HerbertYiga  I tested it in the same way as all the other PRs that I perform, by executing the jetty: run command, with the legacyUI module install.

Comment (dkayiwa) [reply pom.xml:374]: Testing with the reference application would require you to run these modules: https://ci.openmrs.org/build/result/artifactUrlRedirect.action?planKey=REFAPP-OMODDISTRO&buildNumber=9987&artifactName=referenceapplication-addons-snapshot

**Comment by achilep** [comment]
hello @dkayiwa  ,
Screenshot from 2020-06-18 15-13-29

**Pull request merged**